src/Services/QuestionnaireResponseService.php

```diff
@@ -397,20 +397,18 @@ public function saveQuestionnaireResponse(
         }
 
         $version = 1;
-        if (!empty($fhirQuestionnaireOb)) {
```

Comment:
were this value empty we would have hit the `throw` on 383.

Reply:
I agree.